internal/sgbouncewizard/sgbouncewizard.go

```diff
@@ -0,0 +1,237 @@
+package sgbouncewizard
+
+import (
+	"context"
+	"net/http"
+	"log"
+	"encoding/json"
+	"fmt"
+	"github.com/go-chi/chi"
+	"github.com/go-chi/chi/middleware"
+	"github.com/jimmyjames85/bouncecm/internal/db"
+	"github.com/jimmyjames85/bouncecm/internal/models"
+	"github.com/jimmyjames85/bouncecm/internal/config"
+
+	"golang.org/x/crypto/bcrypt"
+
+)
+
+type Server struct {
+	DBClient *db.Client
+}
+
+// NewServer ...
+func NewServer(c config.Configuration) (*Server, error) {
+	client, err := db.NewDB(c)
+	if err != nil {
+		return nil, err
+	}
+
+	return &Server{DBClient: client}, nil
```

Comment:
Before returning the server, I would attempt to ping the db with `err = client.Ping()` and check there was no err, just to make sure we can talk to the backend.